tests/foreman/cli/test_discoveryrule.py

```diff
@@ -172,15 +171,34 @@ def test_positive_create_with_org_loc_id(
         assert class_org.name in rule.organizations
         assert class_location.name in rule.locations
 
+        new_org = Box(target_sat.cli_factory.make_org())
+        new_loc = Box(target_sat.cli_factory.make_location())
+        new_hostgroup = Box(
+            target_sat.cli_factory.make_hostgroup(
+                {'organization-ids': new_org.id, 'location-ids': new_loc.id}
+            )
+        )
+        target_sat.cli.DiscoveryRule.update(
+            {
+                'id': rule.id,
+                'organization-ids': new_org.id,
+                'location-ids': new_loc.id,
+                'hostgroup-id': new_hostgroup.id,
+            }
+        )
+        rule = Box(target_sat.cli.DiscoveryRule.info({'id': rule.id}))
```

Comment:
If not output_format is passed to cli methods, then it returns just normal text, I don't think casting it Box will make any change, please check this and I'd suggest to use  csv/json as output_format instead

Reply:
I checked and its work fine

Reply:
I have updated at someplaces